### PR TITLE
Update ir-breaking-changes and make it visible.

### PR DESCRIPTION
**Pull request by chriseth**

### CODE REVIEW

File: docs/ir-breaking-changes.rst [hunk 1]
```diff
@@ -5,17 +5,40 @@
 Solidity IR-based Codegen Changes
 *********************************
 
-This section highlights the main differences between the old and the IR-based codegen,
-along with the reasoning behind the changes and how to update affected code.
+Solidity can generate EVM bytecode in two different ways:
+Either directly from Solidity to EVM opcodes ("old codegen") or through
+an intermediate representation ("IR") in Yul ("new codegen" or "IR-based codegen").
+
+The IR-based code generator was introduced with an aim to allow
```
Comment (bshastry) [inline docs/ir-breaking-changes.rst]:
```suggestion
The IR-based code generator was introduced with an aim to not only allow
```

**Review by bshastry — COMMENTED** [review]
LGTM. Is there a way to preview the web page to spot formatting errors?

**Comment by chriseth** [comment]
Yes, go to the artifacts of the `b_docs` job: https://898698-40892817-gh.circle-artifacts.com/0/docs-html/ir-breaking-changes.html

**Review by bshastry — DISMISSED** [review]

**Comment by bshastry** [comment]
Layout and text look good to me. Approving.

**Review by ekpyron — DISMISSED** [review]
I haven't checked for exhaustiveness of the docs, but the changes look good!

**Review by cameel — DISMISSED** [review]
I also haven't re-read the whole doc but the changes visible in the diff look reasonable.

**Pull request merged**